### DISCUSSION TITLE
Windhawk GUI does not respond to to Keyboard or Mouse Forward/Back Navigation buttons.  Can this please be added?

### DISCUSSION THREAD

**Discussion by FLskydiver** [ramensoftware/windhawk#188] [General]
Browsing for mods is a bit tedious otherwise, needing to seek out the back arrow.

Support for the Back Key and Arrow Keys would probably be helpful, too, to those who primarily type.

Thank you!

**Comment by m417z** [comment]
Thank you for the feedback, it was brought up recently here, you can follow the issue for progress:
https://github.com/ramensoftware/windhawk/issues/181
I agree that there's a lot of room for improvement, I'll try to improve it in future Windhawk updates.
